nvi-rest/src/test/java/no/sikt/nva/nvi/rest/create/CreateNotesHandlerTest.java

```diff
@@ -0,0 +1,96 @@
+package no.sikt.nva.nvi.rest.create;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import no.sikt.nva.nvi.CandidateResponse;
+import no.sikt.nva.nvi.common.db.Candidate;
+import no.sikt.nva.nvi.common.db.model.DbApprovalStatus;
+import no.sikt.nva.nvi.common.db.model.DbCandidate;
+import no.sikt.nva.nvi.common.db.model.DbNote;
+import no.sikt.nva.nvi.common.service.NviService;
+import no.sikt.nva.nvi.rest.create.CreateNotesHandler;
+import no.sikt.nva.nvi.rest.create.NviNotesRequest;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.AccessRight;
+import nva.commons.apigateway.GatewayResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.zalando.problem.Problem;
+
+public class CreateNotesHandlerTest {
+
+    private Context context;
+    private ByteArrayOutputStream output;
+    private CreateNotesHandler handler;
+    private NviService service;
+
+    @BeforeEach
+    void setUp() {
+        output = new ByteArrayOutputStream();
+        context = mock(Context.class);
+        service = mock(NviService.class);
+        handler = new CreateNotesHandler(service);
+    }
```

Comment:
Instead of mocking nviService, extend LocalTestDatabase and use dynamoDbLocal for testing